swift/files/liberty/swift.conf

```diff
@@ -0,0 +1,182 @@
+[swift-hash]
+
+# swift_hash_path_suffix and swift_hash_path_prefix are used as part of the
+# the hashing algorithm when determining data placement in the cluster.
+# These values should remain secret and MUST NOT change
+# once a cluster has been deployed.
+
+swift_hash_path_suffix = {{ pillar.swift.common.swift_hash_path_suffix }}
+swift_hash_path_prefix = {{ pillar.swift.common.swift_hash_path_prefix }}
+
+# storage policies are defined here and determine various characteristics
+# about how objects are stored and treated.  Policies are specified by name on
+# a per container basis.  Names are case-insensitive.  The policy index is
+# specified in the section header and is used internally.  The policy with
+# index 0 is always used for legacy containers and can be given a name for use
+# in metadata however the ring file name will always be 'object.ring.gz' for
+# backwards compatibility.  If no policies are defined a policy with index 0
+# will be automatically created for backwards compatibility and given the name
+# Policy-0.  A default policy is used when creating new containers when no
+# policy is specified in the request.  If no other policies are defined the
+# policy with index 0 will be declared the default.  If multiple policies are
+# defined you must define a policy with index 0 and you must specify a
+# default.  It is recommended you always define a section for
+# storage-policy:0.
+#
+# A 'policy_type' argument is also supported but is not mandatory.  Default
+# policy type 'replication' is used when 'policy_type' is unspecified.
+{%- for ring in pillar.swift.ring_builder.rings %}
+{%- if ring.get("enabled", True) %}
+
+[storage-policy:{{ loop.index0 }}]
+name = {{ ring.get('policy', {}).get('name', "Policy-"~loop.index0) }}
+{%- for k, v in ring.get('policy', {}).iteritems() %}
```

Comment:
was an intention here to keep formula flexible during initial evolution, subject of change in future versions. Will create an issue for that.